docs/sphinx/walkthrough/turbine.rst

```diff
@@ -50,7 +50,7 @@ the tools directory of the AMR-Wind repo, as follows:
 
 .. code-block:: console
 
-    python <path to amr-wind repo>/tools/calc_inflowoutflow_stats.py -sf <path to precursor directory>/post_processing/abl_statistics14400.nc -ts 7200 -te 9000
+    python <path to amr-wind repo>/tools/calc_inflowoutflow_stats.py -sf <path to precursor directory>/post_processing/abl_statistics14400.nc -ts 7212.5 -te 7800
```

Comment:
```suggestion
    python <path to amr-wind repo>/tools/calc_inflowoutflow_stats.py -sf <path to precursor directory>/post_processing/abl_statistics14400.nc -ts 7212.5 -te 7800
```
let's keep the 7200 because I think it's more intuitive right now. Depending on what you do with the python script, we could change it in the future.